src/routes/home.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
```

Comment:
so the router.js file is a router which you telling it what to do on the route but the this home.js file isn't a route necessarily it's a controller(or handler, but controller seems to be the term used in express) which your router.js calls on a route ('/')